index.bs

```diff
@@ -813,6 +813,10 @@ To <dfn>serialize an [=attribution report=]</dfn> |report|, run the following st
 
 1. Return the [=byte sequence=] resulting from executing [=serialize an infra value to JSON bytes=] on |data|.
 
+Issue: How can we force [=attribution report/event id=] and [=attribution report/trigger data=] to
```

Comment:
Could we directly convert these to strings before placing them in the map?

Reply:
I couldn't find a way to convert a 64-bit integer (or any integer, necessarily) into a string, without relying on a JS function.

Reply:
I think we can copy what fetch does for now: https://fetch.spec.whatwg.org/#serialize-an-integer